apps-rendering/src/renderer.ts

```diff
@@ -10,7 +10,10 @@ import {
 	QandaAtom,
 	TimelineAtom,
 } from '@guardian/atoms-rendering';
-import FigCaption from '@guardian/common-rendering/src/components/figCaption';
+import {
+	default as FigCaption,
+	Variant as FigCaptionVariant,
+} from '@guardian/common-rendering/src/components/figCaption';
```

Comment:
FYI, I think this can also be expressed as:

```ts
import FigCaption, {
	Variant as FigCaptionVariant,
} from '@guardian/common-rendering/src/components/figCaption';
```